mysite/static/css/account/settings.css

```diff
@@ -15,6 +15,11 @@ body.settings,body.missions #main .submodule .body { background-color: #fff; pad
     width: 0%
 }
 
+div.message {
```

Comment:
I'd prefer a CSS selector that is more specific on class hierarchy, and less specific on HTML element names, such as:

`.jGrowl-notification .message`

but I can totally live with this as-is, since jGrowl's is the only `class="message"` in the codebase as far as I can tell.

Verified with:

```
git grep 'class=.message' 
```

and reviewing the output manually.